### ISSUE TITLE
Empty Button in Admin issue

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#769]
**Describe the bug**
using wave extension we are getting 2 empty button errors. when we are inspecting them and fixing them , they are not getting fixed.

**To Reproduce**
1. Go to Admin ( http://localhost:9992/cm/app/?view=workflow&debug=true&pathType=unknown )
2. Run wave chrome extension
3. Go to details and see Errors

**Expected behavior**
There should be no Errors.

**Additional context**
This issue seems to be with some js file, which is generating html code dynamically. need to find the right js and see how to fix it.

**Comment by ramdev1980** [comment]
This bug is related to the upper section of the UI in Admin screen, where we create Site, Asses etc. This upper section is hidden now in Admin.
So this bug is not valid now, hence closing it.
